tests/auth_tests/test_forms.py

```diff
@@ -120,6 +120,19 @@ def test_unicode_username(self):
             self.assertFalse(form.is_valid())
 
     @skipIf(six.PY2, "Python 2 doesn't support unicode usernames by default.")
+    def test_normalize_username(self):
+        ohm_username = 'testΩ'  # U+2126 OHM SIGN
+        data = {
+            'username': ohm_username,
+            'password1': 'pwd2',
+            'password2': 'pwd2',
+        }
+        form = UserCreationForm(data)
+        self.assertTrue(form.is_valid())
+        user = form.save()
+        self.assertNotEqual(user.username, ohm_username)
```

Comment:
@timgraham The best way I can think of is to check the difference between the result of normalize_username and the original one because I can't find exactly what is the result of that method (how they encode that string). 
Since time is short, and it's not a really good way to test this, you can pick up and correct it, I will be happy to learn. Thanks